### PR TITLE
Feature/sof 4369 - TensorFlow GPU Instructions

### PR DESCRIPTION
**Pull request by AcylSilane**

### CODE REVIEW

File: lang/en/docs/jobs-cli/batch-scripts/overview.md [hunk 1]
```diff
@@ -34,3 +34,4 @@ We provide some examples on how to enter the relevant information about jobs in
 
 [^2]: [Maui Scheduler, Official Website](https://en.wikipedia.org/wiki/Maui_Cluster_Scheduler)
 
+[^3]: [TORQUE, Wikipedia](https://en.wikipedia.org/wiki/TORQUE)
```
Comment (AcylSilane) [inline lang/en/docs/jobs-cli/batch-scripts/overview.md:37]: Unrelated change, but I noticed the citation on this page was missing while working on this PR

File: mkdocs.yml [hunk 2]
```diff
@@ -152,6 +152,8 @@ nav:
             - Interpolated Sets:                     tutorials/materials/interpolated-sets.md
             - Molecule on a Surface:                 tutorials/materials/molecule-surface.md
             - Interface:                             tutorials/materials/slabs-interface.md
+        - Python Packages:
```
Comment (timurbazhirov) [inline mkdocs.yml:155]: Probably fits under "General Functionality", no?

Comment (AcylSilane) [reply mkdocs.yml]: Fixed in recent commit

**Comment by AcylSilane** [comment]
Merging per https://exabyte.atlassian.net/browse/SOF-4369?focusedCommentId=41287

**Pull request merged**